pkg/virt-api/rest/subresource_test.go

```diff
@@ -953,14 +981,17 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 				vm.Status.VolumeRequests = existingVolumeRequests
 			}
 
-			patch, err := generateVMVolumeRequestPatch(vm, volumeRequest)
+			generatedPatch, err := generateVMVolumeRequestPatch(vm, volumeRequest)
 			if expectError {
 				Expect(err).To(HaveOccurred())
-			} else {
-				Expect(err).ToNot(HaveOccurred())
+				Expect(generatedPatch).To(Equal([]byte{}))
+				return
 			}
 
-			Expect(patch).To(Equal(expectedPatch))
+			Expect(err).ToNot(HaveOccurred())
+			expectedPatch, err := expectedPatchSet.GeneratePayload()
+			Expect(err).To(Not(HaveOccurred()))
```

Comment:
nit: ToNot() or To(Not())?

Reply:
"ToNot() or To(Not()) that is the question" [cit. Hamlet]

Reply:
Or NotTo()?